python/src/aiconfig/editor/client/src/components/AIConfigEditor.tsx

```diff
@@ -681,13 +681,15 @@ export default function AIConfigEditor({
         promptId,
         cancellationToken,
       });
+      logEventHandler?.("RUN_PROMPT_START");
 
       const onPromptError = (message: string | null) => {
         dispatch({
           type: "RUN_PROMPT_ERROR",
           promptId,
           message: message ?? undefined,
         });
+        logEventHandler?.("RUN_PROMPT_ERROR");
```

Comment:
Should we log the error message in the data as well? And maybe the prompt model

Reply:
I was thinking so too but I think righ now the error logs pass the entire aiconfig. wasn't sure if we want to be logging that since we said we don't want to log the config. Thoughts? 

Should we pass in the message?

Reply:
If the `message` includes the entire aiconfig, we shouldn't log that (and should probably fix to not do that). I  was pretty sure it just returns an error message without config, though

Reply:
going to ship this and then investigate this (and make change as necessary ontop)